package.json

```diff
@@ -28,100 +28,112 @@
   "author": "egdelwonk",
   "license": "MIT",
   "dependencies": {
-    "@slack/client": "^3.10.0",
-    "autoprefixer": "7.1.2",
-    "babel-cli": "6.24.1",
-    "babel-core": "6.25.0",
-    "babel-eslint": "7.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.1",
-    "babel-plugin-transform-react-constant-elements": "6.23.0",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.6",
+    "@slack/client": "^3.13.0",
+    "autoprefixer": "^7.1.4",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.0.1",
+    "babel-jest": "^21.2.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-react-constant-elements": "^6.23.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.8",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "6.23.0",
-    "babel-preset-env": "1.6.0",
-    "babel-preset-react": "6.24.1",
-    "babel-preset-react-hmre": "1.1.1",
-    "babel-preset-stage-1": "6.24.1",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-1": "^6.24.1",
     "bcrypt-nodejs": "^0.0.3",
-    "body-parser": "^1.17.2",
+    "body-parser": "^1.18.2",
     "bookshelf": "^0.10.4",
     "bookshelf-cascade-delete": "^2.0.1",
-    "bulma": "^0.5.1",
-    "chalk": "2.0.1",
+    "bulma": "^0.5.3",
+    "chalk": "^2.1.0",
     "classnames": "^2.2.5",
-    "compression": "^1.7.0",
-    "connect-history-api-fallback": "1.3.0",
+    "compression": "^1.7.1",
+    "connect-history-api-fallback": "^1.3.0",
     "cookie-parser": "^1.4.3",
-    "coveralls": "2.13.1",
-    "css-loader": "0.28.4",
+    "coveralls": "^2.13.1",
+    "css-loader": "^0.28.7",
     "dotenv": "^4.0.0",
-    "enzyme": "2.9.1",
-    "eslint": "4.2.0",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-react": "7.1.0",
-    "eslint-watch": "3.1.2",
-    "express": "^4.15.3",
+    "enzyme": "^3.0.0",
```

Comment:
Hey @egdelwonk Is enzyme working for you? I am getting an error about `enzyme-react-adapter-16`.

Reply:
Here is the docs that pointed me there. 
https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md
Had an issue installing the adapter, but it appears to be working now.